test/framework/src/main/java/org/elasticsearch/test/rest/yaml/section/DoSection.java

```diff
@@ -548,7 +548,9 @@ private static NodeSelector parseVersionSelector(XContentParser parser) throws I
         if (false == parser.currentToken().isValue()) {
             throw new XContentParseException(parser.getTokenLocation(), "expected [version] to be a value");
         }
-        List<VersionRange> skipVersionRanges = SkipSection.parseVersionRanges(parser.text());
+        List<VersionRange> skipVersionRanges = parser.text().equals("current")
```

Comment:
I don't like current here, it was pretty confusing when I looked at it for the first time, but I cannot come up with anything better, and more I look at it, the more it makes sense :smile:. I tried `latest` and `same` but they seem to be even more confusing.

Reply:
`Version.CURRENT` is the name of the constant. But, yeah, I get it. In the yaml context its weird.

Reply:
I'm going to merge this with `current` and hope that we can come up with a better name soon.

Reply:
It makes perfect sense in Version, since there is only one version that we are currently working with.

Reply:
I'm with you that it is weird. But all the alternatives I can think of are confusing too. `latest`, `same`, `this_branch`.